### ISSUE TITLE
digitalocean deploy: can't reach volume quorum

### ISSUE THREAD

**Issue by thypon** [deis/deis#4984]
```
;) deisctl start platform                                                                                  
● ▴ ■
■ ● ▴ Starting Deis...
▴ ■ ●
Storage subsystem...
deis-store-monitor.service: active/running                           
deis-store-daemon.service: active/running                            
deis-store-metadata.service: active/running                          
deis-store-gateway@1.service: active/running                         
deis-store-volume.service: active/running                            
Logging subsystem...
deis-logger.service: active/running                                  
deis-logspout.service: active/running                                
Control plane...
deis-controller.service: active/running                              
deis-registry@1.service: active/running                              
deis-database.service: active/running                                
deis-builder.service: activating/start-post    
```

```
;) deisctl  list                                                                                           
UNIT                            MACHINE                         LOAD    ACTIVE          SUB
deis-builder.service            REDACTED         loaded  activating      start-post
deis-controller.service         REDACTED       loaded  active          running
deis-database.service           REDACTED       loaded  active          running
deis-logger.service             REDACTED       loaded  active          running
deis-logspout.service           REDACTED         loaded  activating      auto-restart
deis-logspout.service           REDACTED       loaded  active          running
deis-logspout.service           REDACTED       loaded  activating      auto-restart
deis-publisher.service          REDACTED         loaded  active          running
deis-publisher.service          REDACTED       loaded  active          running
deis-publisher.service          REDACTED       loaded  active          running
deis-registry@1.service         REDACTED       loaded  active          running
deis-router@1.service           REDACTED       loaded  active          running
deis-router@2.service           REDACTED       loaded  active          running
deis-router@3.service           REDACTED         loaded  active          running
deis-store-daemon.service       REDACTED         loaded  active          running
deis-store-daemon.service       REDACTED       loaded  active          running
deis-store-daemon.service       REDACTED       loaded  active          running
deis-store-gateway@1.service    REDACTED         loaded  active          running
deis-store-metadata.service     REDACTED         loaded  active          running
deis-store-metadata.service     REDACTED       loaded  active          running
deis-store-metadata.service     REDACTED       loaded  active          running
deis-store-monitor.service      REDACTED         loaded  active          running
deis-store-monitor.service      REDACTED       loaded  active          running
deis-store-monitor.service      REDACTED       loaded  active          running
deis-store-volume.service       REDACTED         loaded  active          running
deis-store-volume.service       REDACTED       loaded  deactivating    final-sigkill
deis-store-volume.service       REDACTED       loaded  deactivating    final-sigkill
```

I used the default terraform scripts with 3 digitalocean droplets with 4GB ram each. All test were conducted on SFO1, FRA1, NYC1, without success.

**Comment by carmstrong** [comment]
What's your kernel version? I suspect this is https://github.com/deis/deis/issues/4937 and https://github.com/coreos/bugs/issues/1092

**Comment by thypon** [comment]
i used last coreos images provided by the terraform script

**Comment by thypon** [comment]
```
core@coreos-test ~ $ uname -a
Linux coreos-test 4.3.6-coreos #2 SMP Wed Mar 23 01:17:29 UTC 2016 x86_64 Intel(R) Xeon(R) CPU E5-2650L v3 @ 1.80GHz GenuineIntel GNU/Linux
```

Could be related. This is the kernel contained in the last coreos-stable image inside digitalocean.

**Comment by mischief** [comment]
yes - it is. the upstream ceph bug here is http://tracker.ceph.com/issues/14360, and the workaround appears to be implemented in deis via https://github.com/deis/deis/pull/4985.

**Comment by carmstrong** [comment]
> and the workaround appears to be implemented in deis

:+1: Thanks for the pointer to the workaround, @mischief!

**Comment by thypon** [comment]
Fix alredy merged. I'll wait to close this one when the next release is out.

**Comment by mboersma** [comment]
This should be fixed in Deis v1.13.0.
